### ISSUE TITLE
Arch Site Map View Issue

### ISSUE THREAD

**Issue by emjaymur** [legiongis/hmsf-md#51]
I am having  issues with viewing archaeological sites that I've added to my account.  They show up with the markers, but the overall shape of the site does  not show up, as it does while in the admin account.
![map view arch site](https://user-images.githubusercontent.com/37811747/38832504-c15e81fc-4190-11e8-888d-c9ac9bd211e7.jpg)

**Comment by rosulliv14** [comment]
I noticed this as well, originally this was also happening for the Admin account. In the admin account I went in to the "Map Layer Manager" and switched over the button for "Add to search map by default". That fixed it for the admin account and the other admin level type logins (like for FMSF etc) but not for the state park level logins or scouts.

![arches1](https://user-images.githubusercontent.com/37839712/38875388-40c25e72-4228-11e8-9845-a5de2cfb9a55.JPG)

**Comment by mradamcox** [comment]
@emjaymur, this is not by design per se, but it is the best I was able to do when implementing the new permissions system. Filtering that archaeological site layer in the same way that the actual search results are filtered (in the case of scouts, for example, based on their user name) is very complex and I have yet to figure out the best way to do it. What you have described is the original way I set things up: FL_BAR, FMSF, and admin have access to the layer because those groups are allowed to see all archaeological sites anyway. For all other groups, scouts, state park users, etc, there is no access to that layer.

I realize now that this is perhaps a reason to have the hexagons present, in that they would allow the user to view the extent of all archaeological sites in the search results.
